opencart 1.5.6.4/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+
+services:
+  opencart:
+    build: ./
+    restart: always
+    ports:
+      - 8080:80
+    environment:
+      OPENCART_DB_HOST: opencart_db
```

Comment:
Are those environment variable used anywhere?

Reply:
No, I misunderstood how this part worked. Removed